tket/src/Circuit/basic_circ_manip.cpp

```diff
@@ -33,27 +33,51 @@ namespace tket {
 // this method removes them and removes the vertices
 // from boundaries
```

Comment:
Please correct this comment as it isn't quite accurate any more.

Reply:
done in @cqc-melf
[update comment in void Circuit::remove_blank_wires](https://github.com/CQCL/tket/pull/1481/commits/cc8d6293e567307084a5887029219c795d29e9db)
[cc8d629](https://github.com/CQCL/tket/pull/1481/commits/cc8d6293e567307084a5887029219c795d29e9db)